redis/src/cluster_client.rs

```diff
@@ -33,6 +33,7 @@ struct BuilderParams {
     retries_configuration: RetryParams,
     connection_timeout: Option<Duration>,
     topology_checks_interval: Option<Duration>,
+    client_name: Option<String>,
```

Comment:
minor: you can use `Option<ArcStr>` throughout, and remove the extra allocations.

Reply:
Again, I prefer unity overs asking why some of the params (password, username) are by Option and this one is by ArcStr